Movied/Movied/Scenes/Favorite/FavoriteViewController.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+
+final class FavoriteViewController: UIViewController {
+
+    @IBOutlet private weak var favoriteTitle: UILabel!
+    @IBOutlet private weak var favoriteMovieListTableView: UITableView!
+    
+    private var favoriteViewModel = FavoriteViewControllerViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configViewModel()
+        configView()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        favoriteViewModel.loadFavoriteMovies()
+    }
+}
+//MARK: - Configure UI
+extension FavoriteViewController {
+    private func configViewModel() {
+        favoriteViewModel.reloadTableView = { [weak self] in
+            self?.favoriteMovieListTableView.reloadData()
+        }
+    }
+    
+    private func configView() {
+        favoriteTitle.text = Title.favorite
+        configTableView()
+    }
+    
+    private func configTableView() {
+        favoriteMovieListTableView.separatorStyle = .none
+        favoriteMovieListTableView.delegate = self
+        favoriteMovieListTableView.dataSource = self
+        favoriteMovieListTableView.rowHeight = favoriteMovieListTableView.frame.height / 4
+        favoriteMovieListTableView.registerNib(cellName: FavoriteMovieItemTableViewCell.className)
+    }
+}
+//MARK: - Table View Datasource
+extension FavoriteViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView,
+                   numberOfRowsInSection section: Int) -> Int {
+        return favoriteViewModel.numberOfAllFavoriteMovieCells
+    }
+    
+    func tableView(_ tableView: UITableView,
+                   cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withClass: FavoriteMovieItemTableViewCell.self,
+                                                 for: indexPath)
+        if 0..<favoriteViewModel.numberOfAllFavoriteMovieCells ~= indexPath.item {
+            let movieCellViewModel = favoriteViewModel.getFavoriteMovieCellViewModel(at: indexPath)
+            cell.favoriteMovieImageView.getImageFromURL(APIURLs.Image.original + movieCellViewModel.movieImageURLString)
```

Comment:
Don't access UI of cell at outside of cell. Should pass the data to ViewModel of cell

Reply:
Pass data to cell at init cell's ViewModel should be fine